lib/wallet/src/Cardano/Wallet/Primitive/Types/Tx/SealedTx.hs

```diff
@@ -179,6 +185,18 @@ cardanoTxIdeallyNoLaterThan
     -> InAnyCardanoEra Cardano.Tx
 cardanoTxIdeallyNoLaterThan era = unsafeCardanoTx . ideallyNoLaterThan era
 
+-- | Re-deserialises the bytes 'SealedTx' as a transaction in the provided era
```

Comment:
The whole `SealedTx` ugly and should probably be rethought. Perhaps using something like https://hackage.haskell.org/package/these-1.2/docs/Data-These.html